source/Digitisers/src/DDPlanarDigiProcessor.cc

```diff
@@ -308,10 +348,41 @@ void DDPlanarDigiProcessor::processEvent( LCEvent * evt ) {
           continue; 
         }
       }
+
+      //***************************************************************
+      // Smear time of the hit and apply the time window cut if needed
+      //***************************************************************
       
-      //**************************************************************************
-      // Try to smear the hit but ensure the hit is inside the sensitive region
-      //**************************************************************************
+      // Smearing time of the hit
```

Comment:
```suggestion
      double hitT = simTHit->getTime();
      // Smearing time of the hit
      if (_resT.size() and _resT[0] > 0.0) {
```
indent and close at the end.